src/parameter_entry_class.F90

```diff
@@ -488,7 +493,7 @@ subroutine get_vector_character (this, value, errc)
     type is (character(*))
 #ifdef INTEL_DPD200255963
       if (allocated(value)) deallocate(value)
-      allocate(value(size(v)), source=v)
+      allocate(character(len(v(1))) :: value(size(v)))
```

Comment:
Yeah, this was introduced in the last commit 7c5b24084ea55a3a428d5d3dc54abb95706b6d42, and it's just a hackish workaround, and there still seem to be problems at runtime. We might not include that commit for now, until we have a better patch. The corresponding bug in the gfortran bug tracker is here: https://gcc.gnu.org/bugzilla/show_bug.cgi?id=54070